monai/networks/layers/factories.py

```diff
@@ -274,91 +259,91 @@ def instance_nvfuser_factory(dim):
     return optional_import("apex.normalization", name="InstanceNorm3dNVFuser")[0]
 
 
-Act.add_factory_callable("elu", lambda: nn.modules.ELU)
-Act.add_factory_callable("relu", lambda: nn.modules.ReLU)
-Act.add_factory_callable("leakyrelu", lambda: nn.modules.LeakyReLU)
-Act.add_factory_callable("prelu", lambda: nn.modules.PReLU)
-Act.add_factory_callable("relu6", lambda: nn.modules.ReLU6)
-Act.add_factory_callable("selu", lambda: nn.modules.SELU)
-Act.add_factory_callable("celu", lambda: nn.modules.CELU)
-Act.add_factory_callable("gelu", lambda: nn.modules.GELU)
-Act.add_factory_callable("sigmoid", lambda: nn.modules.Sigmoid)
-Act.add_factory_callable("tanh", lambda: nn.modules.Tanh)
-Act.add_factory_callable("softmax", lambda: nn.modules.Softmax)
-Act.add_factory_callable("logsoftmax", lambda: nn.modules.LogSoftmax)
+Act.add("elu", lambda: nn.modules.ELU)
```

Comment:
thanks for the refactoring! with the new API, I suspect the `lambda: nn.modules.ELU` could be further simplified as `nn.modules.ELU`...

Reply:
Remember though that what's being added here is a callable which the factory will call to return the layer that you want. We could add a `add_type` method which takes the type and does this lambda thing.

Reply:
I added an `add_factory_class` method that does the lambda internally, and simplified the activations you pointed out @wyli. It looks like a lot more could be further simplified now, actually - all the factory additions which only supply a single class. Shall I go ahead and do that?